### ISSUE TITLE
[Bug Report] When using fixed columns and aggregate functions, the horizontal scroll bar in the lower left corner is blocked by aggregate rows.

### ISSUE THREAD

**Issue by XiaoFan1519** [ElemeFE/element#12871]
### Element UI version
2.4.7

### OS/Browsers version
Win10 / Chrome 69.0.3497.100

### Vue version
2.5.17

### Reproduction Link
[https://jsfiddle.net/xtqbpmLe/](https://jsfiddle.net/xtqbpmLe/)

### Steps to reproduce
https://jsfiddle.net/xtqbpmLe/
点击滚动条左边。

### What is Expected?
能点击并拖动滚动条。

### What is actually happening?
无法点击到滚动条。

**Comment by ziyoung** [comment]
Duplicate of #12871
